File: FrontEnd/src/app/reviews/reviewInfo.ts
export class ReviewInfo {
    titulo:string|null;
    comentario:string|null;
    nota:number|null;

    username:string|null;
    videojuego:string|null;
    fechaRegIni:string|null;
    fechaRegFin:string|null;
    notaIni:string|null;
    notaFin:string|null;
    order:string|null;

    [key: string]: any;


    constructor(){
        this.titulo = null;
        this.comentario = null;
        this.nota = null;

        this.username = null;
        this.videojuego = null;
        this.fechaRegIni = null;
        this.fechaRegFin = null;
        this.notaIni = null;
        this.notaFin = null;
        this.order = null;
    }
}

File: reviews_service/src/main/java/upm/tfg/b190222/reviews_service/info/ReviewInfo.java
package upm.tfg.b190222.reviews_service.info;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class ReviewInfo {
     private String titulo;
     private String comentario;
     private Integer nota;

     private String username;
     private String videojuego;
     private String fechaRegIni;
     private String fechaRegFin;
     private String notaMediaIni;
     private String notaMediaFin;
     private String order;
}
